src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,102 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final int RESIZE_FACTOR = 2;
+    private static final double LOAD_FACTOR = 0.75;
+    private Node<K, V>[] buckets;
+    private int size;
+    private int threshold;
```

Comment:
For the future, the current threshold can be calculated based on size and array length, not need to have separate variables for them